ci-operator/step-registry/redhat-performance/benchmark-runner/redhat-performance-benchmark-runner-commands.sh

```
@@ -4,52 +4,80 @@ set -euo pipefail
+    KUBEADMIN_PASSWORD=$(<"/secret/kubeadmin_password")
+    KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD%$'\n'}"
+  fi
   export KUBEADMIN_PASSWORD
-fi
 
-if [[ -s /secret/kubeconfig ]]; then
+  CLUSTER_API=$(ssh ${SSH_ARGS} root@"${CLUSTER_IP}" "grep server /root/.kube/config 2>/dev/null | head -1 | awk '{print \$2}'" || true)
+  CLUSTER_API="${CLUSTER_API%$'\n'}"
+  CLUSTER_API="${CLUSTER_API// /}"
+  if [[ -n "${CLUSTER_API}" ]]; then
+    ssh ${SSH_ARGS} root@"${CLUSTER_IP}" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)